lang/en/docs/tutorials/other/jupyter.md

```diff
@@ -0,0 +1,44 @@
+# Jupyter Notebook
```

Comment:
1. Let's follow the video step by step here too
2. Add a link to exabyte-api-examples repository

Reply:
Done.